.github/actions/build-dotnet/action.yml

```diff
@@ -0,0 +1,25 @@
+﻿name: .NET Build with Static Code Analysis
+description: Builds all .NET solutions in the given directory with static code analysis.
+
+inputs:
+  directory:
+    required: false
+    default: .
+    description: Path to the directory where a solution file can be found. Defaults to ".".
+  verbosity:
+    required: false
+    default: quiet
+    description: Verbosity parameter for dotnet build. Defaults to "quiet".
+  enable-code-analysis:
+    required: false
+    default: "true"
+    description: Indicates whether to enable static code analysis during build. Defaults to "true".
```

Comment:
It's like this in the .NET world, sorry, whether you like it :D.

Reply:
![image](https://user-images.githubusercontent.com/4007293/174458025-cd007331-6937-4b3d-b286-b4068683474a.png)

Reply:
:D

Reply:
This [answer](https://english.stackexchange.com/a/359727) sums it up quite nicely.

Reply:
Or we could just use "if". I'm pretty sure we are software developers.

Maybe I should make an esolang called DotNet that's just C# but you have to write `whether (isTrue) { doSomething(); }` instead of `if (isTrue) { doSomething(); }`. That should also mess with SEO which is always fun. 😈